src/containers/CartPage/index.js

```diff
@@ -0,0 +1,203 @@
+                                        checkButtonRole = true;
+                                    }
+                                })
+                            }
+                            if (checkButtonRole) {
+                                return (
+                                    <RPS.RestaurantItem key={product.id}>
+                                        <RPS.RestaurantItemImage src={product.photoUrl} ></RPS.RestaurantItemImage>
+                                        <RPS.RestaurantItemName>{product.name}</RPS.RestaurantItemName>
+                                        <RPS.RestaurantItemIngredients>{product.description}</RPS.RestaurantItemIngredients>
+                                        <RPS.RestaurantItemPrice>{"R$" + product.price.toFixed(2)}</RPS.RestaurantItemPrice>
+                                        {this.props.restaurantOrder && this.props.restaurantOrder.map(item => {
+                                            if (product.id === item.id) {
+                                                orderValue += (product.price * item.quantity)
+                                                return (
+                                                    <RPS.RestaurantCountItens><RPS.RestaurantCounterItensText>{item.quantity}</RPS.RestaurantCounterItensText></RPS.RestaurantCountItens>
+                                                )
+                                            }
+                                        })}
+                                        <RPS.RestaurantButtomSubIten onClick={() => this.props.delOrder(product.id)}><RPS.RestaurantButtomSubItenText>Remover</RPS.RestaurantButtomSubItenText></RPS.RestaurantButtomSubIten>
+                                    </RPS.RestaurantItem>
+                                )
+                            }
+                        })}
+                    </RPS.Restaurant>
+                    <CPS.TotalContainer>
+                        <CPS.Freight>{"Frete R$" + restaurantDetails.shipping.toFixed(2)}</CPS.Freight>
+                        <CPS.SubTotal>SUBTOTAL</CPS.SubTotal>
+                        <CPS.TotalValue>R${orderValue.toFixed(2)}</CPS.TotalValue>
+                    </CPS.TotalContainer>
+
+                    <CPS.PayMethodLabel>Forma de Pagamento</CPS.PayMethodLabel>
+
+                    <CPS.LineBreak />
+
+                    <CPS.PayMethodContainer>
+                        <RadioGroup name="paymentMethod" value={this.state.paymentMethod} onChange={this.handleChangePaymentMethod}>
+                            <FormControlLabel value="money" control={<GreenRadio />} label="Dinheiro" />
+                            <FormControlLabel value="creditcard" control={<GreenRadio />} label="Cartao" />
+                        </RadioGroup>
+                    </CPS.PayMethodContainer>
+
+                    <CPS.ConfirmButton onClick={()=>this.handleSendOrder()}>Confirmar</CPS.ConfirmButton>
```

Comment:
Quando temos métodos/funções que não recebem parâmetros, podemos passar a referência diretamente para o `onClick`: `onClick={this.handleSendOrder}`, a vantagem de fazer dessa forma é que vc poupa a criação de uma função extra (`() =>`) a cada render.